MediaBrowser.Controller/Collections/CollectionCreatedEventArgs.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using MediaBrowser.Controller.Entities.Movies;
+
+namespace MediaBrowser.Controller.Collections
+{
+    /// <summary>
+    /// Object storing the arguments for the <c>CollectionCreated</c> event.
```

Comment:
You should use `<see cref="..."/>` instead of `<c>` here

Reply:
This was adapted from a suggestion from @Bond-009 I believe. What's the argument for one situation vs the other?

Reply:
IMO `<see cref="..."/>` should be used whenever possible, it creates a link to the referenced property, class, etc. in the generated documentation, is picked up when renaming variables in the IDE and provides a warning if the name is invalid. I think <c> is just use to format the text as code, similar to `like this` in markdown

Reply:
I think this fix might have gotten reverted as well

Reply:
Sorry nvm, I was looking at the wrong commit in GitHub